### DISCUSSION TITLE
AWS Marketplace AMI Removed?

### DISCUSSION THREAD

**Discussion by npazo** [requarks/wiki#4186] [Help / Questions] [answered]
Anyone know why the AWS Marketplace entry for Wiki.js was removed? I can't find anything about it on the main site and the documentation still lists it as an install option.

**Comment by AVTPJ** [comment]
Same here, can someone please supply a new AMI?

**Comment by NGPixel** [comment, marked as answer]
The AMI was removed by AWS because the base image was too old. Unfortunately the current script to generate these images is currently broken and I didn't have time to look into it yet. I suggest using the [Ubuntu install guide](https://docs.requarks.io/install/ubuntu) with a standard Ubuntu instance in the meantime. The result will be exactly the same as what you would get with the AMI.

**Reply by quarfie** [reply]
Perhaps this information should be added to the docs at https://docs.requarks.io/install/aws
